scipy/signal/__init__.py

```diff
@@ -113,6 +113,11 @@
    invres        -- Inverse partial fraction expansion for analog filter.
    invresz       -- Inverse partial fraction expansion for digital filter.
    BadCoefficients  -- Warning on badly conditioned filter coefficients.
+   sosfreqz      -- Digital filter frequency response for SOS format filter.
+                    -- Alias for `freqz_sos`.
+
+``sosfreqz`` is an alias of `freqz_sos` for backward compatibility
+(should not be used in new code).
```

Comment:
I don't think we need these in the module `__doc__` (which then shows up on the website). I think it's good enough to have a note in `sosfreqz`'s `__doc__` instead.

Reply:
Because `sosfreqz` is not in the index, the 0.19.0 release notes can't build properly.
I'm going to try dig this a bit. Does someone knows how to get rid of this warning?
```
/home/circleci/repo/doc/source/release/0.19.0-notes.rst:117: WARNING: py:obj reference target not found: scipy.signal.sosfreqz
```